tests/tools/test_isort_works.py

```diff
@@ -155,3 +156,55 @@ def test_isort_trailing_semicolon(tmp_notebook_with_trailing_semicolon: Path) ->
         '+    "    pass;"\n'
     )
     assert result == expected
+
+
+def test_old_isort_separated_imports(tmp_test_data: Path) -> None:
+    """
+    Check isort works when a notebook has imports in different cells.
+
+    This test would fail if we didn't pass --treat-comment-as-code '# %%'.
+
+    Parameters
+    ----------
+    tmp_test_data
+        Temporary copy of test data.
+    """
+    notebook = tmp_test_data / "notebook_with_separated_imports_other.ipynb"
+
+    with open(notebook) as handle:
```

Comment:
How about using Path.read_text()?
Or
How about using `os.path.getmtime` to see if the notebook was modified if the intent is to check if the notebook content got changed?